core/transformation/pathfinders/version_1.py

```diff
@@ -0,0 +1,491 @@
+        id_vars=["Intervention theme", "Outcome", "Unit of measurement"],
+        var_name="Reporting Period",
+        value_name="Amount",
+    )
+    start_dates = melted_df["Reporting Period"].map(
+        lambda x: PF_REPORTING_PERIOD_TO_DATES[", ".join(x.split(", ")[:-1])]["start"]
+    )
+    end_dates = melted_df["Reporting Period"].map(
+        lambda x: PF_REPORTING_PERIOD_TO_DATES[", ".join(x.split(", ")[:-1])]["end"]
+    )
+    actual_forecast = melted_df["Reporting Period"].map(lambda x: "Actual" if "Actual" in x else "Forecast")
+    return {
+        "Outcome_Ref": pd.DataFrame(
+            {
+                "Outcome Name": outcomes,
+                "Outcome Category": outcome_categories,
+            }
+        ),
+        "Outcome_Data": pd.DataFrame(
+            {
+                "Higher Frequency": [pd.NA] * len(melted_df),
+                "Project ID": [pd.NA] * len(melted_df),
+                "Programme ID": [programme_id] * len(melted_df),
+                "Outcome": melted_df["Outcome"],
+                "UnitofMeasurement": melted_df["Unit of measurement"],
+                "GeographyIndicator": [pd.NA] * len(melted_df),
+                "Amount": melted_df["Amount"],
+                "Actual/Forecast": actual_forecast,
+                "Start_Date": start_dates,
+                "End_Date": end_dates,
+            }
+        ),
+    }
+
+
+def _risk_register(
+    df_dict: dict[str, pd.DataFrame],
+    programme_name_to_id_mapping: dict[str, str],
+) -> pd.DataFrame:
+    """
+    Populates `risk_register` table:
+        project_id              - from "Project ID" in the transformed DF
+        programme_junction_id   - assigned during map_data_to_models based on "Programme ID" in the transformed DF
+        event_data_blob         - includes "Risk Name", "Risk Category", "Short Description", "Pre-mitigated Impact",
+                                  "Pre-mitigated Likelihood" and "Mitigations" from the transformed DF
+    """
+    organisation_name = df_dict["Organisation Name"].iloc[0, 0]
+    programme_id = programme_name_to_id_mapping[organisation_name]
+    risks = df_dict["Risks"]
+    return pd.DataFrame(
+        {
+            "Programme ID": [programme_id] * len(risks),
+            "Project ID": [pd.NA] * len(risks),
+            "RiskName": risks["Risk name"],
+            "RiskCategory": risks["Category"],
+            "Short Description": risks["Description"],
+            "Full Description": [pd.NA] * len(risks),
+            "Consequences": [pd.NA] * len(risks),
+            "Pre-mitigatedImpact": risks["Impact score"],
+            "Pre-mitigatedLikelihood": risks["Likelihood score"],
+            "Mitigations": risks["Mitigations"],
+            "PostMitigatedImpact": [pd.NA] * len(risks),
+            "PostMitigatedLikelihood": [pd.NA] * len(risks),
+            "Proximity": [pd.NA] * len(risks),
+            "RiskOwnerRole": [pd.NA] * len(risks),
+        }
+    )
```

Comment:
As an extension to my point above about hardcoding empty columns.. This seems even less necessary, as all these fields are to be stored in a JSONB field. I think we should let the serialisation handle missing values in the `/download` extract - what do you think?